src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.CoreRT.cs

```diff
@@ -332,8 +360,31 @@ public object GetOrRegisterObjectForComInstance(IntPtr externalComObject, Create
                 throw new NotImplementedException();
 
             object? wrapperMaybeLocal = wrapperMaybe;
-            retValue = null;
-            throw new NotImplementedException();
+            retValue = impl.CreateObject(externalComObject, flags);
```

Comment:
Should this do a lookup in the list of active objects first before creating the object?

Reply:
I believe I raise this question regarding RCW creation. If caching is responsibility of runtime, then yes, I have to lookup in the list of active objects first.

Reply:
Does that mean that in addition to `ConditionalWeakTable<object, NativeObjectWrapper> _rcwTable` I need separate `ConcurrentDictionary<IntPtr, WeakReference<object>> _rcwCache` ?

Reply:
Yes, something like that. It needs to match this from CoreCLR: https://github.com/dotnet/runtime/blob/01b7e73cd378145264a7cb7a09365b41ed42b240/src/coreclr/vm/interoplibinterface.cpp#L708

Reply:
Still missing?

Reply:
I think this is covered already.

Reply:
I think this is already implemented.